it/connectors/GetSavingsIncomeDataTysConnectorISpec.scala

```diff
@@ -100,9 +100,10 @@ class GetSavingsIncomeDataTysConnectorISpec extends PlaySpec with WiremockSpec {
       "DES returns an empty 200" in {
         stubGetWithResponseBody(url, OK, returnedEmpty.toString())
         val result = await(connector.getSavingsIncomeData(nino, specificTaxYear))
-        val expectedResult = ErrorModel(INTERNAL_SERVER_ERROR, ErrorBodyModel.parsingError)
+        val expectedResult = Right(SavingsIncomeDataModel(None, None, None))
+          //ErrorModel(INTERNAL_SERVER_ERROR, ErrorBodyModel.parsingError)
```

Comment:
Was this meant to be removed?

Reply:
Yes as it no longer returns a parsing error with all optional properties, returns the empty model like on line 103 instead.

Reply:
Yes it was meant to be removed my bad.